### ISSUE TITLE
Accessibility: unable to open popup on voice messages on doubletap with longpress

### ISSUE THREAD

**Issue by beqabeqa473** [signalapp/Signal-Android#11519]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Unable to open popup actions of voice messages by doubletapping message with longpress.

### Steps to reproduce
- open conversation with a person, try to doubletap with longpress voice message with talkback turned on.

**Actual result:** Nothing is happening.
**Expected result:** Popup actions should be openned while talkback is on.
### Device info

**Device:** Samsung Galaxy s21
**Android version:** 11.0.0
**Signal version:** Latest

**Comment by alex-signal** [comment]
I'm not quite sure what is expected here. Do you have an example of what you'd expect to see?

**Comment by beqabeqa473** [comment]
if i doubletap a voice message with longpress, popup with actions such
as forward, delete, and so one should appear.

it is not happening when talkback is turned on.

On 8/3/21, Alex Hart ***@***.***> wrote:
> I'm not quite sure what is expected here. Do you have an example of what
> you'd expect to see?
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/signalapp/Signal-Android/issues/11519#issuecomment-891830076


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Comment by alex-signal** [comment]
Could you try to press outside of the bounds of the voice message bubble? It's likely that you're pressing down on the seek bar which is consuming the touch event. Or are you saying that even in this case it is not functioning?

**Comment by beqabeqa473** [comment]
No, i am not pressing on seekbar, i am pressing on voice message,
label of that element is time when it was send/received. i also tried
to press on play/pause button, i know it shouldn't work, but anyyway
nothing works.

On 8/3/21, Alex Hart ***@***.***> wrote:
> Could you try to press outside of the bounds of the voice message bubble?
> It's likely that you're pressing down on the seek bar which is consuming the
> touch event. Or are you saying that even in this case it is not
> functioning?
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/signalapp/Signal-Android/issues/11519#issuecomment-891966625


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili
